zfit/minimizers/minimizer_nlopt.py

```diff
@@ -0,0 +1,576 @@
+#  Copyright (c) 2021 zfit
+import collections
+import copy
+import math
+from typing import Optional, Dict, Union, Callable, Mapping
+
+import nlopt
+import numpy as np
+
+from .baseminimizer import BaseMinimizer, minimize_supports, NOT_SUPPORTED, print_minimization_status
+from .evaluation import LossEval
+from .fitresult import FitResult
+from .strategy import ZfitStrategy
+from .termination import EDM, CRITERION_NOT_AVAILABLE, ConvergenceCriterion
+from ..core.parameter import set_values
+from ..settings import run
+from ..util.exception import MaximumIterationReached
+
+
+# class NLopt(BaseMinimizer):
```

Comment:
This commented out class is tried to be imported in zfit/minimize.py. which leads to an error.

Reply:
yes, forgot to push the newest commit, fixed now